### PR TITLE
Adds plaintext FTP capture server

### PR DESCRIPTION
**Pull request by tomsteele**

### CODE REVIEW

File: pkg/flamingo/ftp.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	listener, err := net.Listen("tcp", fmt.Sprintf("%s:%d", c.BindHost, c.BindPort))
+	if err != nil {
+		return err
+	}
+	c.listener = listener
+	go startFTP(c)
+	return nil
+}
+
+func startFTP(c *ConfFTP) {
```
Comment (hdm) [inline pkg/flamingo/ftp.go]: startFTP needs some way to break out of this on shutdown

Comment (tomsteele) [reply pkg/flamingo/ftp.go]: Change `continue` to `break`?

**Comment by hdm** [comment]
Will tweak a few things post-merge, thanks!

**Comment by hdm** [comment]
Post-merge I changed the `for { }` to `for !c.IsShutdown() { }` to resolve this, thanks again!

**Pull request merged**